<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QGLViewer" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTextures">
    <property name="title">
     <string>Textures</string>
    </property>
    <addaction name="actionTCMask"/>
    <addaction name="actionChoose_Texture"/>
   </widget>
   <widget class="QMenu" name="menuTeam_Colours">
    <property name="title">
     <string>Team Colours</string>
    </property>
    <addaction name="actionTexture_Frames"/>
    <addaction name="separator"/>
    <addaction name="actionCTCMask"/>
    <addaction name="separator"/>
    <addaction name="actionFixed_Pipeline"/>
    <addaction name="actionShaders"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTextures"/>
   <addaction name="menuTeam_Colours"/>
  </widget>
  <action name="actionTransformWidget">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Transform</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionUVEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UVEditor</string>
   </property>
   <property name="toolTip">
    <string>UVEditor</string>
   </property>
  </action>
  <action name="actionTexture_Frames">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Texture Frames</string>
   </property>
  </action>
  <action name="actionCTCMask">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>TCMask</string>
   </property>
   <property name="toolTip">
    <string>TCMask</string>
   </property>
  </action>
  <action name="actionFixed_Pipeline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fixed Pipeline</string>
   </property>
  </action>
  <action name="actionShaders">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shaders</string>
   </property>
  </action>
  <action name="actionTCMask">
   <property name="text">
    <string>Choose TCMask...</string>
   </property>
  </action>
  <action name="actionChoose_Texture">
   <property name="text">
    <string>Choose Texture...</string>
   </property>
  </action>
  <action name="actionConfig">
   <property name="text">
    <string>Config</string>
   </property>
   <property name="toolTip">
    <string>Config</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QGLViewer</class>
   <extends>QWidget</extends>
   <header>QGLViewer/qglviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
